entry.go

```diff
@@ -419,3 +426,7 @@ func (entry *Entry) sprintlnn(args ...interface{}) string {
 	msg := fmt.Sprintln(args...)
 	return msg[:len(msg)-1]
 }
+
+func golang1_14() bool {
+	return strings.HasPrefix(runtime.Version(), goVersion1_14)
```

Comment:
If this fix is still needed on go1.14 final, it's cleaner to use build tags for this instead of determining go version at runtime; have a file that has a `!go1.14` build-tag  for older go versions, and an implementation in a file with a `go1.14` build-tag for the current version.

See https://github.com/docker/go-connections/pull/68/files for an example

Reply:
👍  Updated, please review again.